content/00.front-matter.md

```diff
@@ -27,14 +27,14 @@ on {{manubot.date}}.
 
 {## Template for listing authors ##}
 {% for author in manubot.authors %}
-+ **{{author.name}}**<br>
```

Comment:
Feel free to reopen this pull request if you'd like to see how the reference you added renders in the HTML output.

If you want to modify the authors in a manuscript, you have to add them to the list in [`content/metadata.yaml`](https://github.com/manubot/try-manubot/blob/main/content/metadata.yaml).  The lines here in `content/00.front-matter.md` are a template that loops over every author in the metadata file.